src/masonite/orm/grammar/mysql_grammar.py

```diff
@@ -146,12 +146,12 @@ def create_column_length(self):
     def unique_constraint_string(self):
         return "CONSTRAINT {clean_column}_unique UNIQUE ({clean_column})"
```

Comment:
@corentinalcoy @josephmancuso 
Does here is just add a *args to receive more than one columns?

Reply:
No wait, Joe is looking the way the schema module is done. I think we have a problem...